website/docs/replicalimits.md

```diff
@@ -135,7 +135,7 @@ spec:
 Usage
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper-library/master/library/general/replicalimits/samples/replicalimits/example_allowed.yaml
```

Comment:
I don't think you need to update these since these should be the example deployments, not the constraints. as you can see https://open-policy-agent.github.io/gatekeeper-library/website/replicalimits#examples

Reply:
I regenerated the website and the change was reverted.